app/questionnaire/questionnaire_schema.py

```diff
@@ -707,6 +717,9 @@ def get_first_block_id_for_section(self, section_id: str) -> str | None:
     def get_blocks(self) -> Iterable[ImmutableDict]:
         return self._blocks_by_id.values()
 
+    def get_repeating_blocks(self) -> dict[str, ImmutableDict]:
+        return self._repeating_blocks_by_id
```

Comment:
Yeah I did think this when adding, I opted for returning the blocks by id so that anything else in the future (routing/path finding etc etc) can make use of this, the result can be transformed in whichever way the caller requires.

Reply:
Will change to only returning IDs tho

Reply:
Now returns just the ids 👍